.travis.yml

```diff
@@ -1,6 +1,16 @@
-sudo: false
```

Comment:
I thought it was the default. In looking at the documentation, it is the default for projects that began using Travis on 1/1/2015.

It's unneeded for this project. Should we include it anyway, though?